### PR TITLE
AKS Updates

### PR DESCRIPTION
**Pull request by ibersanoMS**
- Add subnet for Windows subnet 
- Monitoring on AKS
- Added a third nodepool (linux)

### CODE REVIEW

File: Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md [hunk 1]
```diff
@@ -32,19 +32,30 @@ Using the same PowerShell session from the previous step, update the state file
...
+### Exercise the User Nodepool
+The reference implementation creates 3 nodepools: a Linux system nodepool, a Linux user nodepool and a Windows nodepool. The Linux user nodepool is not actively used in this implementation, but the intended uses for this nodepool include hosting ingress controllers, 
+
+Try out the following command to exercise this nodepool
+```
+az aks command invoke \
+  --resource-group $AKS_RESOURCE_GROUP \
+  --name $AKS_CLUSTER_NAME \
+  --command "kubectl get pods -n kube-system"
+```
```
Comment (ckittel) [inline Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md:57]:
~~~suggestion
### Try the user nodepool

The reference implementation creates three nodepools: a Linux system nodepool, a Windows nodepool, and a Linux user nodepool. The Linux user nodepool is not actively used in this implementation, but the intended uses for this nodepool include hosting ingress controllers, or any other workload in the cluster that requires a Linux host -- which is common when running Windows clusters.

This example uses AKS command invoke, which creates a temporary pod to invoke a command on the cluster.  This common management command creates a transient pod in the cluster, and that pod requires a Linux node pool to be available.

```PowerShell
az aks command invoke \
  --resource-group $AKS_RESOURCE_GROUP \
  --name $AKS_CLUSTER_NAME \
  --command "kubectl get pods -n kube-system"
```
~~~

File: Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md [hunk 2]
```diff
@@ -32,19 +32,30 @@ Using the same PowerShell session from the previous step, update the state file
 Once the files are updated, deploy using Terraform Init, Plan and Apply.
```
Comment (ckittel) [inline Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md:32]: We still have `wnp_count = true` above -- I thought we got rid of that?

**Review by ckittel — CHANGES_REQUESTED** [review]

File: Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md [hunk 3]
```diff
@@ -32,19 +32,30 @@ Using the same PowerShell session from the previous step, update the state file
...
 If you get an error about changes to the configuration, go with the `-reconfigure` flag option.
 
+### Exercise the User Nodepool
+The reference implementation creates 3 nodepools: a Linux system nodepool, a Linux user nodepool and a Windows nodepool. The Linux user nodepool is not actively used in this implementation, but the intended uses for this nodepool include hosting ingress controllers, 
+
+Try out the following command to exercise this nodepool
+```
+az aks command invoke \
+  --resource-group $AKS_RESOURCE_GROUP \
+  --name $AKS_CLUSTER_NAME \
```
[the remark this answers is not in the document]
Comment (ibersanoMS) [reply Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md:58]: Added to environment variables section

**Review by dcint — COMMENTED** [review]
Approved all

**Pull request merged**